include/envoy/secret/secret_manager.h

```diff
@@ -37,6 +42,20 @@ class SecretManager {
    */
   virtual TlsCertificateConfigProviderSharedPtr createInlineTlsCertificateProvider(
       const envoy::api::v2::auth::TlsCertificate& tls_certificate) PURE;
+
+  /**
+   * Finds and returns a dynamic secret provider associated to SDS config. Create
+   * a new one if such provider does not exist.
+   *
+   * @param config_source a protobuf message object contains SDS config source.
+   * @param config_name a name that uniquely refers to the SDS config source
+   * @param secret_provider_context context that provides components for creating and initializing
+   * secret provider.
+   * @return the dynamic TLS secret provider.
```

Comment:
Nit: `@return TlsCertificateConfigProviderSharedPtr the dynamic TLS secret provider.`

Reply:
Done. Thanks.